.github/workflows/mcp-javascript-ci.yml

```
@@ -0,0 +1,57 @@
+name: mcp-javascript-ci
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)